sdk/metricsadvisor/Azure.AI.MetricsAdvisor/README.md

```diff
@@ -225,21 +225,21 @@ Console.WriteLine($"Data feed status: {createdDataFeed.Status.Value}");
 Console.WriteLine($"Data feed created time: {createdDataFeed.CreatedTime.Value}");
 
 Console.WriteLine($"Data feed administrators:");
-foreach (string admin in createdDataFeed.Administrators)
+foreach (string admin in createdDataFeed.AdministratorsEmails)
 {
     Console.WriteLine($" - {admin}");
 }
 
 Console.WriteLine($"Metric IDs:");
 foreach (DataFeedMetric metric in createdDataFeed.Schema.MetricColumns)
 {
-    Console.WriteLine($" - {metric.MetricName}: {metric.MetricId}");
+    Console.WriteLine($" - {metric.Name}: {metric.Id}");
```

Comment:
no concern on this PR, more curious about the rename from `metricId` to `id` I have something similar in translator and we have been debating if we should rename it to `id` and that's it. In UX study we didn't find enough data that `<ovbject>id` was confusing so I am unsure what is best :)

Reply:
I think both are ok. It's just that keeping the `Metric` prefix is redundant given this is a `DataFeedMetric` instance. We've been doing the same in lots of models in Metrics Advisor in all languages. To tell you the truth, I think no one really wondered if the change was worth the trouble.

Reply:
yeah.. same feeling.
Thank you!